src/components/Button/Button.tsx

```diff
@@ -74,7 +74,7 @@ const PureButton = React.forwardRef<HTMLButtonElement | HTMLAnchorElement, Butto
         {
             view = 'normal',
             size = 'm',
-            pin,
+            pin = 'round-round',
```

Comment:
Shouldn't be this change considered as breaking?

Reply:
Seems ok